scipy/stats/_hypotests.py

```diff
@@ -692,23 +692,25 @@ def barnard_exact(table, alternative="two-sided", pooled=True, num_it=3):
         * 'greater': one-sided
 
     pooled : bool, optional
-        Whether to compute score statistic with pooled variance or
-        unpooled (Wald statistic)
+        Whether to compute score statistic with pooled variance (Student's
+        t-test) or unpooled variance (Welch's t-test). Default is ``True`` :
+        Statistic test is computed using pooled variance
```

Comment:
```suggestion
        Statistic test is computed using pooled variance.
```

Reply:
Update in b0121ffdad593d9903b9d0b664de974a22a896f3